prebuilt-ui/ios-webview/ios-webview/WebViewPrebuilt/ViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ViewController.swift
+//  WebViewPrebuilt
+//
+//  Created by Daily on 10/06/2021.
+//
+
+import UIKit
+import WebKit
+
+class ViewController: UIViewController {
+    private var webView: WKWebView? = nil
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let webView = Self.makeWebView()
+        self.webView = webView
+        
+        self.view.addSubview(webView)
+        webView.translatesAutoresizingMaskIntoConstraints = false
+        webView.leadingAnchor.constraint(equalTo: self.view.leadingAnchor).isActive = true
+        webView.trailingAnchor.constraint(equalTo: self.view.trailingAnchor).isActive = true
+        webView.topAnchor.constraint(equalTo: self.view.topAnchor).isActive = true
+        webView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor).isActive = true
+
+        let url = URL(string: "[YOUR DAILY ROOM URL HERE]");
+        
+        webView.load(URLRequest(url: url!));
+    }
+
+    private static func makeWebView() -> WKWebView {
+        let webpagePreferences = WKWebpagePreferences()
+        webpagePreferences.allowsContentJavaScript = true
+        webpagePreferences.preferredContentMode = .mobile
+
+        let configuration = WKWebViewConfiguration()
+
+        // Make sure the webview's `<audio>` elements are
+        // allowed to auto-play inline:
+        configuration.allowsInlineMediaPlayback = true
+        // … and without user interaction:
+        configuration.mediaTypesRequiringUserActionForPlayback = []
+        
+        configuration.defaultWebpagePreferences = webpagePreferences
+
+        // We don't intend to ever show the webview, so we give it a zero-size:
```

Comment:
This comment is obsolete, as unlike party-line we _do_ intend to show the webview.